pkgs/applications/audio/mp3gain/default.nix

```diff
@@ -11,6 +11,12 @@ stdenv.mkDerivation {
 
   sourceRoot = ".";
 
+  patchFlags = "-p2";
+
+  patches = [
+    ./patches/0001-fix-security-bugs.patch
```

Comment:
1. There aren't any other patches there right? No need to put it in a directory if there's only 1 patch, IMO.
2. Please add a comment saying where the patch came from, or ideally fetch it with `fetchpatch`. Also, is it pending for inclusion upstream? Will it be included in the next release - answers to these should be in a comment.

Reply:
Thank you @doronbehar, I've replaced the copy of the patch from openSuSE with the one from Gentoo because I couldn't find a plain-text version there.

The information on if it will be included upstream is not known to me. Development is very slow on that project.